frappe/www/contact.py

```diff
@@ -22,38 +22,17 @@ def get_context(context):
 	return out
 
 
-max_communications_per_hour = 1000
-
-
 @frappe.whitelist(allow_guest=True)
-def send_message(subject="Website Query", message="", sender=""):
-	if not message:
-		frappe.response["message"] = "Please write something"
-		return
-
-	if not sender:
-		frappe.response["message"] = "Email Address Required"
-		return
+@rate_limit(limit=1000, seconds=60 * 60, methods=["POST"])
```

Comment:
Remove method?

That weirdly only limits on specified method.

Reply:
done via https://github.com/frappe/frappe/pull/20542/commits/4af4d8e5de9debe9d638ab28bf41a853973c4ebf